react-locate-contractor/src/Pages/PersonalPage.tsx

```diff
@@ -1,11 +1,42 @@
-import React from "react";
+import { useState, useEffect } from 'react';
+import { useOktaAuth } from '@okta/okta-react';
 
 const PersonalPage = () => {
+    <h3 id="personal">Personal</h3>
+    const { authState, oktaAuth } = useOktaAuth();
+    const [userInfo, setUserInfo]: [userInfo: any, setUserInfo: any] = useState(null);
+
+    useEffect(() => {
+        if (!authState || !authState.isAuthenticated) {
+            // When user isn't authenticated, forget any user info
+            setUserInfo(null);
+        } else {
+            oktaAuth.getUser().then((info: any) => {
+                setUserInfo(info);
+            }).catch((err) => {
+                console.error(err);
+            });
+        }
+    }, [authState, oktaAuth]); // Update if authState changes
+
+    if (!userInfo) {
+        return (
+            <div>
+                <p>Uh oh, you are not logged in! To access you're personal account you must be logged in...</p>
+            </div>
+        );
+    }
+
```

Comment:
This element seems to repeat a lot and can probably be made into a re-usable component